pkg/storage/unified/sql/sqltemplate/dialect_mysql.go

```diff
@@ -26,6 +31,11 @@ type backtickIdent struct{}
 var standardFallback = standardIdent{}
 
 func (backtickIdent) Ident(s string) (string, error) {
```

Comment:
@diegommm -- I wonder if we actually want different function for table identifiers vs concrete strings?

Reply:
Yeah, this seems like very strange behavior. I agree that concrete strings should be quoted separately from identifiers.

https://dev.mysql.com/doc/refman/8.4/en/identifiers.html

For MySQL, backticks are always acceptable for identifiers, and single-quotes are always acceptable for strings.

Double-quotes are used for identifiers in ANSI mode, and for strings in "normal" mode. It seems like the safest path is to never use double-quotes as a quoting character for MySQL.